commit 87d8f8e666b91e2bc63f3c24212b7ad4253e1458 (refs/changes/90/4690/3)
Author: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date:   2018-08-17T15:13:22+02:00 (1 year, 2 months ago)
    
    OS-6913 ubuntu 18.04 hangs during boot on bhyve
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin
old mode 100644
new mode 100755
index 0687c892..a90f7760
Binary files a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin and b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin differ
diff --git a/overlay/generic/usr/share/bhyve/uefi-rom.bin b/overlay/generic/usr/share/bhyve/uefi-rom.bin
old mode 100644
new mode 100755
index cfefe0be..ba2214a4
Binary files a/overlay/generic/usr/share/bhyve/uefi-rom.bin and b/overlay/generic/usr/share/bhyve/uefi-rom.bin differ
